app/config/FrontendAppConfig.scala

```diff
@@ -48,6 +48,8 @@ class FrontendAppConfig @Inject() (configuration: Configuration, servicesConfig:
 
   val lockedUrl: String = s"$departureHubUrl/cannot-open"
 
+  val draftDeparturesUrl: String = s"$hubUrl/draft-declarations"
```

Comment:
Change this to
`val draftNotAvailableUrl: String = s"$departureHubUrl//draft-declarations/not-available"`